### PR TITLE
find-jira-issue 0.0.1

### PR DESCRIPTION
**Pull request by jimmithy**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [X] __I will not move an already shared step version's tag to another commit__
- [X] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [X] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [X] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [X] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/find-jira-issue/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      The input used to locate the jira commit message.
+
+      For example, the git commit message or branch name.
+    is_expand: true
+    is_required: true
+    summary: Where should we look for jira issues
+    title: Input
+    value_options: []
+outputs:
+- JIRA_ISSUE: null
```
Comment (koral--) [inline steps/find-jira-issue/0.0.1/step.yml:38]: It should be rather named JIRA_ISSUE_LIST

File: steps/find-jira-issue/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- FIND_ISSUE_INPUT: $BITRISE_GIT_MESSAGE
+  opts:
+    description: |
+      The input used to locate the jira commit message.
+
+      For example, the git commit message or branch name.
+    is_expand: true
+    is_required: true
+    summary: Where should we look for jira issues
+    title: Input
```
Comment (koral--) [inline steps/find-jira-issue/0.0.1/step.yml:35]: It should be more descriptive perhaps. `Input` tells nothing IMHO.

File: steps/find-jira-issue/0.0.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- FIND_ISSUE_INPUT: $BITRISE_GIT_MESSAGE
```
Comment (koral--) [inline steps/find-jira-issue/0.0.1/step.yml:26]: input name should use lower_snake_case and should not include word input

Comment (jimmithy) [reply steps/find-jira-issue/0.0.1/step.yml]: changed to `find_issue_content`

File: steps/find-jira-issue/0.0.1/step.yml (hunk 2 above)
Comment (jimmithy) [reply steps/find-jira-issue/0.0.1/step.yml]: Updated to `Text Content`

File: steps/find-jira-issue/0.0.1/step.yml (hunk 1 above)
Comment (jimmithy) [reply steps/find-jira-issue/0.0.1/step.yml]: Update to `JIRA_ISSUE_LIST`

**Comment by trapacska** [comment]
@jimmithy thanks for the PR 🚀

**Pull request merged**